frontend/src/components/Shared/VideoPlayer/VideoPlayer.jsx

```
@@ -0,0 +1,229 @@
+    <iframe 
+      width="100%" 
+      height="100%" 
+      src={createYouTubeEmbedUrl(youtubeId)}
+      title={title || "Video"}
+      style={{ border: 0 }}
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+      allowFullScreen={false}
+      className={`z-20 ${videoLoaded ? 'opacity-100' : 'opacity-0'}`}
+      id="youtube-player"
+      onLoad={handleVideoLoad}
+    ></iframe>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)